### ISSUE TITLE
Crashlytics found an invalid API key: null.

### ISSUE THREAD

**Issue by Giampa** [redsolution/xabber-android#854]
Could you please help me with this issue?
I've cloned the repository, init and update the submodules (the respective folders are correctly populated),
I've chosen the following flavours:

xabber: devDebug
emojicon: debug
MemorizingTrustManager: debug

No way to build the project on Android Studio latest stable version, I receive:

> Crashlytics found an invalid API key: null. 
> Check the Crashlytics plugin to make sure that the application has been added successfully! 
> Contact support@fabric.io for assistance.

**Comment by panpansh** [comment]
just register account on fabric.io and add the keys to fabric.properties

**Comment by Str4tocaster** [comment]
@Giampa Seems like something changed in crashlytics side. 
I am fixed this issue in develop branch.

You need to clone new project from branch **develop**.
Then use **OPEN** flavour for build Xabber.
If project will have file **fabric.properties** delete it.

Hope, i helped you.

**Comment by Giampa** [comment]
@Str4tocaster thanks for your reply. I've tried cloning your develop brach.
I've some problem with MemorizingTrustManager module.

`git submodule init`
`git submodule update`

But the folder with that submodule remain with a single .iml file (emojicon is ok). I didn't have the same issue with master branch. I copied the content of the master module in the develop folder, should that be ok? Many thanks in advance.
